### DISCUSSION TITLE
Kodum doğru fakat

### DISCUSSION THREAD

**Discussion by Baris5455** [19/classroom#63] [C]
Hocam kodumu kendi terminalimde çalıştırınca istediğiniz gibi çıktı veriyor ama githuba yükleyince sıkıntı çıkıyor Strong basması gerekn yerde invalid basıyor ne yapmalıyım?

![resim_2023-04-12_223235674](https://user-images.githubusercontent.com/123635074/231565116-21b4c270-ef3a-451c-b003-e91dbe199c43.png)

![resim_2023-04-12_223314563](https://user-images.githubusercontent.com/123635074/231565227-cb51d9b4-327a-459f-9d11-b8786f8bed9b.png)

**Comment by roktas** [comment]
Bu yasal bir soru değil. :-)  Cevap kısaca: "çözmelisin"  Sondaki boşluklar dahil herşeye dikkat ederek.
